appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -43,6 +43,9 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
                     data: rt
                 };
             });
+
+            // filters out Node 4.x and 6.x as they are EOL
+            runtimeItems = this.filterRuntimes(runtimeItems, /node\|(4|6)\./);
```

Comment:
nit: I almost always throw an `i` flag at the end of a regexp to handle weird casing